ci/run_tests/requirements.txt

```diff
@@ -1 +1 @@
-netifaces==0.11.0
+scapy==2.5.0
```

Comment:
I would avoid using scapy, since we should not use a big library for just a small function.

Reply:
Let me try to find other alternative methods.